### PR TITLE
Cleanup repository

### PR DESCRIPTION
**Pull request by jenniferboedker**
cleans up the repo and readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,42 @@
 # Sample command-line Tool
```
Comment (sven1103) [inline README.md:1]: needs to be centered as well. also, a small subtitle in italic is missing. please check the template readme again as reference

Comment (sven1103) [reply README.md:1]: I also think the the title is not how it is supposed to be: "Sample command-line tool"? Maybe "Sample Status CLI", "A command-line tool to...."

**Review by sven1103 — CHANGES_REQUESTED** [review]
Some minor change requests

File: README.md (hunk 1 above)
Comment (jenniferboedker) [reply README.md]: very good point, is addressed

File: .github/release.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - JohnnyQ5
```
Comment (Steffengreiner) [inline .github/release.yml:6]: Adapt to current template 
```suggestion
      - JohnnyQ5
      - github-actions
```

File: README.md [hunk 3]
```diff
@@ -1,20 +1,39 @@
...
-## Description
+## How to run
 
-## How to Install
+Create a runable version of this code with maven and java 8:
 
-Download the application and run it with:
+```
+> mvn clean package
+
```
Comment (Steffengreiner) [inline README.md:22]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,20 +1,39 @@
...
-> java -jar sample-status-updater-cli.jar -h
+|-target
+|---sample-status-updater-cli-1.0.0.jar
+|---...
+```
+
+Run the application with:
+
+```
+> java -jar sample-status-updater-cli-1.0.0.jar -h
```
Comment (Steffengreiner) [inline README.md:36]: You can also remove the newlines in the code block here

File: pom.xml [hunk 5]
```diff
@@ -13,7 +13,7 @@
   <artifactId>sample-status-updater-cli</artifactId>
   <version>1.4.0-SNAPSHOT</version>
```
Comment (Steffengreiner) [inline pom.xml:14]: NIT: Why is this a Snapshot version? 😕

**Review by Steffengreiner — DISMISSED** [review]
Nearly There 👍. 
Also it would make sense imo to add picocli to the license section of the readme. 
https://github.com/remkop/picocli/blob/main/LICENSE
Also one could think about removing the groovy-checkstyle workflow since it's not included in the template

File: pom.xml (hunk 5 above)
Comment (jenniferboedker) [reply pom.xml:14]: probably because maven removed the snapshot already automated here? 🤷🏼 
do you want me to update the versioning here?

Comment (sven1103) [reply pom.xml:14]: leave it, the CI takes over. no need to touch this anymore

**Review by sven1103 — APPROVED** [review]

**Comment by sven1103** [comment]
Please provide a meaning full commit message, since the PR title is not very explicit :)

**Pull request merged**